cockatrice/src/deck_loader.cpp

```diff
@@ -200,9 +200,19 @@ void DeckLoader::saveToStream_DeckZoneCards(QTextStream &out, const InnerDecklis
 QString DeckLoader::getCardZoneFromName(QString cardName, QString currentZoneName)
 {
     CardInfo *card = db->getCard(cardName);
-    if(card && card->getIsToken())
+    if (card && card->getIsToken())
         return DECK_ZONE_TOKENS;
 
     return currentZoneName;
 }
 
+QString DeckLoader::getCompleteCardName(QString cardName)
```

Comment:
This doesn't explain why you wrote a mutating method that implies its a getter. Compared to `getCardZoneFromName` from the same header file which does not mutate parameters.